src/app/board/board.component.html

```diff
@@ -1,10 +1,47 @@
-<div id="board_topWorkItems" class="container-fluid col-md-12 col-sm-12 col-xs-12">
+<div id="board_topWorkItems" class="container-fluid col-md-12 col-sm-12 col-xs-12 boardWrapper">
   <div class='board container-cards-pf'>
```

Comment:
Follow one rule, either single quote or double quote. For HTML double quote is preferable.